src/resmom_win/start_exec.c

```diff
@@ -2579,10 +2589,10 @@ start_process(task *ptask, char **argv, char **envp)
 	}
 	else {
 #endif
-		/* Normal batch job, single node, write straight to files
+		/* Normal batch job, single node, task with nodemux, write straight to files
 		 * For interactive job, do not write to files.
 		 */
-		if (!is_interactive_job && (pjob->ji_numnodes == 1) && (open_std_out_err(pjob) == -1)) {
+		if (!is_interactive_job && (pjob->ji_numnodes == 1 || nodemux == 1) && (open_std_out_err(pjob) == -1)) {
```

Comment:
Please add parenthesis around (pjob->ji_numnodes == 1) and (nodemux == 1). We could also use (nodemux != 0) because we're using an unsigned int to act as a boolean.